src/interactivity/win32/UiaTextRange.hpp

```diff
@@ -29,6 +29,7 @@ Author(s):
 class UiaTextRangeTests;
 #endif
 
+#define INVALID_ID 0

```

Comment:
@ebudai INVALID_ID should be defined as a constexpr instead of a macro.  Take a look at Cpp Core guideline [ES.31](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es31-dont-use-macros-for-constants-or-functions)